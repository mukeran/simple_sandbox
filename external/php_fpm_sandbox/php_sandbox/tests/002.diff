001+ Fatal error: Uncaught Error: Call to undefined function php_sandbox_test1() in /root/simpleIDS/filters/webshell/php/sandbox/php_sandbox/tests/002.php:2
002+ Stack trace:
001- The extension php_sandbox is loaded and working!
002- NULL
003+ #0 {main}
004+   thrown in /root/simpleIDS/filters/webshell/php/sandbox/php_sandbox/tests/002.php on line 2