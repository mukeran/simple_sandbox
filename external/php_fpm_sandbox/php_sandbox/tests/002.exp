The extension php_sandbox is loaded and working!
NULL