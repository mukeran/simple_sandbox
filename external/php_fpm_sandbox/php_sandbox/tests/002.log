
---- EXPECTED OUTPUT
The extension php_sandbox is loaded and working!
NULL
---- ACTUAL OUTPUT
Fatal error: Uncaught Error: Call to undefined function php_sandbox_test1() in /root/simpleIDS/filters/webshell/php/sandbox/php_sandbox/tests/002.php:2
Stack trace:
#0 {main}
  thrown in /root/simpleIDS/filters/webshell/php/sandbox/php_sandbox/tests/002.php on line 2
---- FAILED
