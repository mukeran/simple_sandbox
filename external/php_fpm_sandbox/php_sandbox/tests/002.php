<?php
$ret = php_sandbox_test1();

var_dump($ret);
?>
