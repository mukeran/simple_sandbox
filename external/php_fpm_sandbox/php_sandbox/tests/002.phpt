--TEST--
php_sandbox_test1() Basic test
--SKIPIF--
<?php
if (!extension_loaded('php_sandbox')) {
	echo 'skip';
}
?>
--FILE--
<?php
$ret = php_sandbox_test1();

var_dump($ret);
?>
--EXPECT--
The extension php_sandbox is loaded and working!
NULL
