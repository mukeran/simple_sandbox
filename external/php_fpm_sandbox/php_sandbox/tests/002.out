Fatal error: Uncaught Error: Call to undefined function php_sandbox_test1() in /root/simpleIDS/filters/webshell/php/sandbox/php_sandbox/tests/002.php:2
Stack trace:
#0 {main}
  thrown in /root/simpleIDS/filters/webshell/php/sandbox/php_sandbox/tests/002.php on line 2